Finder/FinderFactory.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle\Finder;
+
+use Symfony\Component\Finder\Finder;
+
+class FinderFactory
```

Comment:
``Finder`` is stateful so on every call to ``AbstractFileExtractor::extractFiles`` which calls to ``AbstractFileExtractor::extractFromDirectory`` i need to create a new finder because I can not reuse an existing one and the only one way to do that is to use a factory otherwise I need to call the constructor of ``Finder`` inside the class which is so bad and make it untestable.